cfgov/jinja2/v1/_includes/molecules/half-width-link-blob.html

```diff
@@ -8,18 +8,18 @@
 
    Create a half-width-link-blob molecule when given:
 
-   settings: An object with the following options for settings.
+   settings:         An object with the following options for settings.
 
-    settings.heading: A string containing title of module.
+   settings.heading: A string containing title of module.
 
-    settings.content: A string containing the content  to display below the
-                      heading (typically a paragraph).
+   settings.content: A string containing the content  to display below the
```

Comment:
This should probably be updated to `body` to match the other macros. Can be done outside this PR.